src/environment.rs

```diff
@@ -62,7 +62,7 @@ pub fn verify_prefix_location_unchanged(environment_dir: &Path) -> miette::Resul
 }
 
 /// Create the prefix location file.
-/// Give it the environment path to place it.
+/// This file is needed for `conda run -p .pixi/envs/<env>` to work.
```

Comment:
Is this true? We actually don't expect `conda` to do anything with this. This is done as a copy protection mechanism. 
If you miss interpreted it just remove it, as my comment was also unusual for our code base.
```suggestion
```

Reply:
Oops, the comment belongs to the function below